bin/dev/entities.php

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env php
+<?php
+/**
+ * @copyright Copyright (C) 2020, Friendica
+ *
+ * @license GNU APGL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Used to check/generate entities for the Friendica codebase
+ *
+ */
+
+require dirname(__DIR__) . '/../vendor/autoload.php';
+
+/**
+ * Custom file printer with tab indention and one line between methods
+ */
+class FriendicaPhpPrinter extends \Nette\PhpGenerator\Printer
+{
+	protected $linesBetweenMethods = 1;
+}
+
+// replaces digits with their names
+function digitToText(string $name) {
+	$name = str_replace('0', 'zero_', $name);
+	$name = str_replace('1', 'one_', $name);
+	$name = str_replace('2', 'two_', $name);
+	$name = str_replace('3', 'three_', $name);
+	$name = str_replace('4', 'four_', $name);
+	$name = str_replace('5', 'five_', $name);
+	$name = str_replace('6', 'six_', $name);
+	$name = str_replace('7', 'seven_', $name);
+	$name = str_replace('8', 'eight_', $name);
+	$name = str_replace('9', 'nine_', $name);
+	return $name;
+}
+
+// Replaces underlines ("_") with camelCase notation (for variables)
+function camelCase($str) {
```

Comment:
There is a camelcase function in PHP: https://www.php.net/manual/de/function.ucwords.php